.circleci/config.yml

```diff
@@ -161,7 +161,7 @@ jobs:
 
   checking:
     docker:
-      - image: 'cimg/ruby:3.1.1-node'
```

Comment:
I think this should reference `cimg/ruby:3.1-node` - ie _not_ specify the patch level - that way we automatically benefit from patch level updates without doing any accidental ruby version bumps